README.md

```diff
@@ -1,21 +1,75 @@
 # Knowledge Pack
 ----
-This pack is targeted for Cribl LogStream users with all levels of experience.  Every time we solve interesting use cases our intent is to add them to this knowledge pack.
+This Pack is designed for Cribl LogStream users at all levels of experience, and maintained by Cribl Solution Engineers. Every time we solve an interesting use case, our intent is to add it to this Knowledge Pack.
+ 
+For legibility, the Knowledge Pack is set up according to a simple principle: 
 
-# Got a Problem? 
+**The Route name matches the Pipeline name matches the Sample name.**
+
+
+# What's in the Pack? 
 ---
-```
-We will solve it, put it in the pack and add you to our contributors list!
-```
+
+The LogStream techniques explained in the Pack include how to:
+
+* Manipulate time using `strptime` and `strftime`.
+* Split, trim, slice, and join data streams.
+* Extract fields without losing data.
+* Turn a JSON string into an object literal.
+* Validate JSON data against a schema.
+* Tag and route data using lookup tables, tags, and output routers.
+* Use the Mask Function to decode data.
+* Use the Mask Function to convert hex to numbers, and use the `C.Decode.hex()` function.
+* Apply a Mask Function (md5) to any values in the object literal, using multiple wildcards.
+* Process Cisco ASA events, using a Lookup Function to drop events, or to return a regex for extracting fields.
+* Convert a UTF-16 dynamic-length array into a string, using Spread syntax.
+* Extract exception and `calling_method` from a stack trace, using split and array references.
+
+# What You Need (Prerequisites)
+---
+* A insatiable thirst for knowledge.
 
 # What to Expect
 ---
 * Knowledge of LogStream
 * Power of the Pipeline
 
-## Requirements
+# Got a Problem?
+---
+And don't see it addressed here? We will solve it, put it in the Pack, and add you to our Contributors list!
+
+
+# Using the Pack
 ---
-A insatiable thirst for knowledge.
+The Knowledge Pack is easiest to use like this:
+
+## Adjust the UI
+---
+When you first open the Knowledge Pack, the Routes tab should be selected. Use the column selector (<img valign="middle" width="24" height="18" src="https://github.com/abetones/cribl-knowledge-pack/blob/main/blob/kp-selector.png?raw=true">) to display the Description column and hide the Filter and Pipeline/Output columns. This lets you see and compare the Routes' Descriptions.
```

Comment:
@bdalpe @dmaislin-cribl 

The absolute paths I used for the images obviously need to be changed, since they point at my fork. Lines 48, 50, and 54.

Reply:
Will do!